### ISSUE TITLE
Payment received on Paytabs but it is Pending on Woocommerce

### ISSUE THREAD

**Issue by yuriohz** [paytabscom/paytabs-woocommerce#8]
Hello,

There is a new issue that we are facing which is that the Payment is stuck on pending in Woocommerce although it was received in Paytabs, what would be the possible issue?

We are on WP version :  5.4.2
WooCommerce Credit Card Payment Gateway - PayTabs Plugin version: 2.3

**Comment by wajihkm** [comment]
Hi @yuriohz ,

Could you please try our latest plugin?
And please delete any previous version you already installed.

The problem you mention could be because the customer redirected to payment page, enter the payment information, then click Pay, but for some reason he did not redirect back to your website (maybe he close the browser)

Best regards.

**Comment by qudsiyya** [comment]
Hello, 
I am facing a similar issue. After the payment on paytab, the user is redirected to the cart page, which is still full. The user gets a payment pending email, whereas the payment goes through and the order is placed.
